Tests/DefaultIlpClientTest.swift

```diff
@@ -13,7 +13,7 @@ final class DefaultIlpClientTest: XCTestCase {
         )
 
         // WHEN the balance is requested
-        guard let balance: Org_Interledger_Stream_Proto_GetBalanceResponse =
+        guard let balance: AccountBalance =
```

Comment:
you actually don't need to type this. You can just let the type be inferred with:
`guard let balance = try? ...`